pcdsdevices/ccm.py

```diff
@@ -773,6 +773,41 @@ def inverse(self, real_pos: namedtuple) -> namedtuple:
         return self.PseudoPosition(energy=energy)
 
 
+class CCMEnergyWithACRStatus(CCMEnergyWithVernier):
+    """
+    CCM energy motor and ACR beam energy request with status.
+    Note that in this cae vernier indicates any ways that ACR will act on the
+    photon energy request. This includes the Vernier, but can also lead to
+    motion of the undulators or the K.
+
+    Parameters
+    ----------
+    prefix : str
+        The PV prefix of the Alio motor, e.g. XPP:MON:MPZ:07A
+    hutch : str, optional
+        The hutch we're in. This informs us as to which vernier
+        PVs to write to. If omitted, we can guess this from the
+        prefix.
+    acr_status_sufix : str
+        Prefix to the SIOC PV that ACR uses to report the move status.
+        For HXR this usually is 'AO805'.
+    """
+    vernier = FCpt(BeamEnergyRequest, '{hutch}',
+                   acr_status_suffix='AO805',
```

Comment:
How do I use the acr_status_suffix here instead of hardcoding the 'AO805'?

Reply:
It looks like `FormattedComponent` only allows you to format the suffix.  `DynamicDeviceComponent` also doesn't quite fit the bill.  I don't see a Component type in ophyd or pcdsdevices that lets you format an arbitrary kwarg, but I feel like we must have done something like this before.
Maybe @ZLLentz has better memory than I do?

Reply:
There's a argument from `Component` called `add_prefix` that defaults to `("suffix", "write_pv")`, which is used to determine which string arguments we should prepend with the prefix. This is re-used in `FormattedComponent` to determine which string arguments should be formatted. So, you could do:
```
    vernier = FCpt(BeamEnergyRequest, '{hutch}',
                   acr_status_suffix='{acr_suffix}',
                   add_prefix=('suffix', 'acr_status_suffix')

    def __init__(self, *args, **kwargs):
        self.acr_suffix = 'AO805'
```

Reply:
Ah thanks, that's a neat backdoor.  (or at least something I've never seen in the Component class)